packages/docs-site/src/components/_ShapeProps.js

```diff
@@ -1,5 +1,6 @@
 import { makeCanvas, shapedefs } from "@penrose/core";
 import React from "react";
+import seedrandom from "seedrandom";
```

Comment:
Thanks for fixing this! Weird that it didn't already show up in CI or something?